pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -350,12 +349,7 @@ func (c *Controller) MCSServices() []model.MCSServiceInfo {
 		mcsService.ClusterSetVIP = si.clusterSetVIP
 	}
 
-	out := make([]model.MCSServiceInfo, 0, len(outMap))
```

Comment:
I think this was intentionally a pointer so the import and export loops would modify the same thing. We lost that.